examples/asr/conf/ssl/nest/nest_fast-conformer.yaml

```diff
@@ -2,22 +2,22 @@
 #
 # Here are the recommended configs for different variants of FastConformer, other parameters are the same as in this config file.
 #
-#  +--------------+---------+---------+----------+----------------+--------------+------------+
-#  | Model        | d_model | n_heads | n_layers |conv_kernel_size| weight_decay |  xscaling  |
-#  +==============+=========+========+===========+================+==============+============+
-#  | Small  (14M) |   176   |    4   |    16     |        9       |     0.0      |    True    |
-#  +--------------+---------+--------+-----------+----------------+--------------+------------+
-#  | Medium (32M) |   256   |    4   |    16     |        9       |     1e-3     |    True    |
-#  +--------------+---------+--------+-----------+----------------+--------------+------------+
-#  | Large (120M) |   512   |    8   |    17     |        9       |     1e-3     |    True    |
-#  +--------------+---------+--------+-----------+----------------+--------------+------------+
-#  | XLarge (616M)|   1024  |    8   |    24     |        9       |     1e-3     |    False   |
-#  +--------------+---------+--------+-----------+----------------+--------------+------------+
-#  | XXLarge(1.2B)|   1024  |    8   |    42     |        9       |     1e-3     |    False   |
-#  +--------------------------------------------------------------+--------------+------------+ 
-
-
-name: "NEST-FastConformer-SSL"
+#  +--------------+---------+---------+----------+----------------+--------------+------------+---------+
+#  | Model        | d_model | n_heads | n_layers |conv_kernel_size| weight_decay |  xscaling  | use_bias|
+#  +==============+=========+========+===========+================+==============+============+=========+
+#  | Small  (14M) |   176   |    4   |    16     |        9       |     0.0      |    True    |   True  |
```

Comment:
Have we ever tried just disabling this by default for all model sizes? Something to try in future models

Reply:
yeah we should try disabling it by default in in the future